waterdata/utils.py

```diff
@@ -0,0 +1,49 @@
+from io import StringIO
+from urllib.parse import urljoin
+
+import pandas as pd
+import requests as r
+
+
+def get_water_services_data(hostname, url_path):
+    """
+    Make a request to USGS water services.
+
+    :param str hostname: Scheme and hostname of the target service
+    :param str url_path: Path
+    :return: Water services data (in RDB format), request status code, request status code description
```

Comment:
Can we call this something more generic (maybe get_nwis_data). Since we pass in the url, this function's purpose is really to set the status code and status_reason to what we want for errors.

Reply:
That function is generic enough that I'm considering just calling it `execute_get_request`.